File: C#/RemoveVowels.cs
public class Solution
{
    //O(n)
    //O(1)
    public string RemoveVowels(string s)
    {
        if (string.IsNullOrWhiteSpace(s)) return "";

        var result = new List<string>();

        for (int i = 0; i < s.Length; i++)
        {
            if (s[i] != 'a' && s[i] != 'e' && s[i] != 'i' && s[i] != 'o' && s[i] != 'u')
            {
                result.Add(s[i].ToString());
            }
        }

        return string.Join("", result);
    }
}

File: Go/removeVowels.go
package main

//O(N)
//O(1)
func removeVowels(s string) string {
	if len(s) == 0 {
		return ""
	}

	result := ""

	for _, char := range s {
		if char != 'a' && char != 'e' && char != 'i' && char != 'o' && char != 'u' {
			result += string(char)
		}
	}

	return result
}
